engine.go

```diff
@@ -146,6 +148,7 @@ func (e *RTCEngine) configure(res *livekit.JoinResponse) error {
 	if err != nil {
 		return err
 	}
+	e.lossyDC.OnMessage(e.handleLossyDataPacket)
```

Comment:
we should hook up reliableDC to this as well.

Reply:
and also rename the function. to the client, a message is received is the same regardless. it's the sender's choice whether they'd like reliable delivery or not.

Reply:
I've handle it for realiable too and renamed to `handleDataPacket`